### PR TITLE
fix doctests

### PR DESCRIPTION
**Pull request by vchuravy**

### CODE REVIEW

File: docs/src/index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ CurrentModule = Enzyme
 
 # Enzyme
 
-Documentation for [Enzyme.jl](https://github.com/wsmoses/Enzyme.jl), the Julia bindings for [Enzyme](https://github.com/wsmoses/enzyme).
+Documentation for [Enzyme.jl](https://github.com/EnzymeAD/Enzyme.jl), the Julia bindings for [Enzyme](https://github.com/EnzymeADenzyme).
```
Comment (vchuravy) [inline docs/src/index.md:7]:
```suggestion
Documentation for [Enzyme.jl](https://github.com/EnzymeAD/Enzyme.jl), the Julia bindings for [Enzyme](https://github.com/EnzymeAD/enzyme).
```

Comment (vchuravy) [inline docs/src/index.md:7]:
```suggestion
Documentation for [Enzyme.jl](https://github.com/EnzymeAD/Enzyme.jl), the Julia bindings for [Enzyme](https://github.com/EnzymeAD/enzyme).
```

File: src/Enzyme.jl [hunk 2]
```diff
@@ -798,18 +779,16 @@ of this is the transpose of the Forward [`jacobian`](@ref)
...
-using Enzyme
-
 f(x) = [x[1]*x[2], x[2]]
 
 grad = jacobian(Reverse, f, [2.0, 3.0])
 
 # output
 
-(([3.0, 2.0], [0.0, 1.0]),)
+([3.0, 2.0],)
```
Comment (vchuravy) [inline src/Enzyme.jl:788]: @wsmoses that this didn't work worries me. Should this be `jacobian(Reverse, f, [2.0, 3.0]; n_outs=Val(2))`

Comment (wsmoses) [reply src/Enzyme.jl:788]: It should absolutely be n_outs=2 and that should also absolutely NOT have a default value. It needs to be precisely the length of the returned array (here 2).

Comment (vchuravy) [reply src/Enzyme.jl:788]: We will need to present a better error message to users then. Keyword values are supposed to have a default value

Comment (wsmoses) [reply src/Enzyme.jl:788]: Make it a required arg? I just was worried that having two args would be confusing/lead to mis ordering

**Pull request merged**